src/kubernetes_api/swagger.clj

```diff
@@ -149,11 +149,42 @@
                                            [[verb operation]])))
                                (into {}))))))
 
+(defn default-paths [paths]
+  {"/apis/" (get-in paths ["/apis/"])
+   "/api/" (get-in paths ["/api/"])})
+
+(defn filter-api
+  "Returns the paths that starts with the given api and version
+   e.g. /apis/{api}/..."
+  [api paths]
+  (into {}
+        (filter #(string/starts-with? (first %) (str "/apis/" api)) paths)))
+
+(defn from-apis
+  "Returns the default paths and the paths for the given apis"
+  [apis paths]
+  (let [api-paths (mapv #(filter-api % paths) apis)]
+    (into (default-paths paths)
+          api-paths)))
```

Comment:
I think we are not supporting filtering core API groups. Is that intentional?
In Kubernetes, we have resources like Pod that has no group `apiVersion: v1`, these are legacy and under `/api/<version>` instead of `/apis/<group>/<version>`.. 

I think we should be able to do:
```
{:apis ["v1", "apps", "extensions.apps/v1"]}
```

We might support this too
```
{:apis [""]}
```

Reply:
Not intentional. 
Agreed. I'll make the changes.

Reply:
Agreed. Not intentional, I'll make the change to filter the apis. The only thing that I noticed is in the `core/client` function, there's an association of the api versions and core api versions, that's why the [default-paths](https://github.com/nubank/k8s-api/pull/17/files#diff-c69cbbc6dfa8a55a33b06777250ef2ebdaed56ebc9044f8cc067e7baa073c2beR152) `api` and `apis` are always added in the api list.

https://github.com/nubank/k8s-api/blob/master/src/kubernetes_api/core.clj#L52-L55

```
 (assoc k8s
        ::api-group-list (internals.martian/response-for k8s :GetApiVersions)
        ::core-api-versions (internals.martian/response-for k8s :GetCoreApiVersions))))
```

Reply:
✔️ https://github.com/nubank/k8s-api/pull/17/commits/3661314d8cb86de58cb626db90ab51ee4b66579e